### PR TITLE
Restore IconLayer Example

### PR DESCRIPTION
**Pull request by Pessimistress**
Using data from https://data.nasa.gov/Space-Science/Meteorite-Landings/gh4g-9sfh
@ericsoco @heshan0131 @apercu

screen shot 2017-05-26 at 10 19 29 am

### CODE REVIEW

**Comment by shaojingli** [comment]
Nice! Please change this line... other than that it's good to go

screen shot 2017-05-26 at 10 46 37 am

File: demo/src/static/workers/meteorites-decoder.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+let count = 0;
+
+onmessage = function(e) {
+  const lines = e.data.text.split('\n');
+
+  lines.forEach(function(line) {
+    if (!line) {
+      return;
+    }
+    if(!coordinates) {
```
Comment (Apercu) [inline demo/src/static/workers/meteorites-decoder.js]: missing space

File: demo/src/static/workers/meteorites-decoder.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    flush();
+    postMessage({action: 'end'});
+  }
+};
+
+function flush() {
+  postMessage({
+    action: 'add',
+    data: result,
+    meta: {count: count}
```
Comment (Apercu) [inline demo/src/static/workers/meteorites-decoder.js]: can't we pass `result.length` instead of managing our own counter?

Comment (Pessimistress) [inline demo/src/static/workers/meteorites-decoder.js]: For this dataset yes. The counter was put in place for large datasets that needs flushing multiple times.

**Comment by ericsoco** [comment]
\o/
who needs movies when you've got meteorites?

**Pull request merged**